src/css/custom.scss

```diff
@@ -121,6 +121,9 @@
     padding: 0;
     margin: var(--ifm-navbar-item-padding-vertical)
       var(--ifm-navbar-item-padding-horizontal);
+    @media screen and (max-width: 1033px) {
+      margin: 0.25rem 0.64rem;
```

Comment:
The fix for this issue isn't hardcoding different margins, so this shouldn't be accepted.

Reply:
Can you help me out ? 
I tried to find where these variables are defined but I am not able to find them